### ISSUE TITLE
Use LSB configuration variant by default

### ISSUE THREAD

**Issue by kamil-certat** [certtools/intelmq#2349]
The debian package, what is our main distribution format, patches IntelMQ source code to force LSB (/etc/intelmq) paths: https://github.com/certtools/intelmq/tree/develop/debian/patches

It looks like we're also recommending those paths in our documentation. I'd like to propose to move them as default configuration variant (probably in IntelMQ 4).

**Comment by gethvi** [comment]
Yes please!

**Comment by sebix** [comment]
Do you intend to keep the `INTELMQ_ROOT` variable to allow (multiple) installations in `/opt`?

**Comment by kamil-certat** [comment]
Of course, I just want to apply the patch https://github.com/certtools/intelmq/blob/develop/debian/patches/fix-intelmq-paths.patch in the main code – without any other changes. The control through the environment variable is really useful, and I use it for my development environment. I just find it useful, if the default path would play together with what we recommend and say in docs.

**Comment by sebix** [comment]
You may also ask on the mailing list for more opinions from the community and the users/devs.

The documentation and default configuration need adaptions too.

**Comment by kamil-certat** [comment]
Mail sent :ok_hand:

**Comment by kamil-certat** [comment]
No responses in the mailing list, positive reactions to the issue - I think we could go with it, right?

**Comment by sebix** [comment]
Yes, I agree. Let's go for it
